test/processors/concerns/http_files_test.rb

```diff
@@ -21,15 +21,25 @@ def logger
     WebMock.allow_net_connect!
   }
 
-  let(:uri) { URI.parse('http://test.prx.org/test/file.mp2') }
+  let(:http_uri) { URI.parse('http://test.prx.org/test/file.mp2') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.